### PR TITLE
Portfolio desktop design

### PR DESCRIPTION
**Pull request by Paskab012**
## In this Desktop design, I was able to perform operations such as :
- Writing a good Readme file for portfolio-desktop-version
- Very well style the code using media queries, flexbox, and grid
- Setup Linters and detect problems 
- To make a good view for the design

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -275,14 +288,14 @@ <h3 class="about-myself-connect">LET'S CONNECT</h3>
         method="POST"
       >
         <div class="all-components">
-          <div class="tonic-content">
+          <div class="tonic-content-contact">
```
Comment (lily-coder) [inline index.html:291]: - [x] [REQUIRED] Please ensure the "Contact Me" section matches the original version by adding the padding to the name, email and textarea inputs as in the original Figma design. 
- [x] [REQUIRED] Please also center the paragraph starting with "If you have an application..." to match the original version.

**Review by lily-coder — CHANGES_REQUESTED** [review]
Hi @KABIKA681 , 

Good job so far!🎉🎉🎉

To Highlight! 🟢
✔️ Good job with the linter checks
✔️ Correct Github flow was used

There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

- [x] [REQUIRED] Please also consider capitalizing your commit messages as that makes your repo more professional. To see how you can make your work more professional you can visit this [link](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md)

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 2]
```diff
@@ -275,14 +288,14 @@ <h3 class="about-myself-connect">LET'S CONNECT</h3>
...
             <p>
               If you have an application you are intersted in developing, a
               feature that you need built or a project that needs codiing, I'd
               love to help with it.
             </p>
-            <div class="tonic-courses">
+            <div class="tonic-courses-about">
               <input
                 type="text"
                 placeholder="Mack donald"
```
Comment (medaminedev66) [inline index.html:301]: - [x] Good job so far but your mobile design still needs some work to be done there. I examined your app using the browser dev tools and the layout of the contact section doesn't fit the mobile screen, please, Kindly fix the mobile version of the contact section. 
- Your design:
image

**Review by medaminedev66 — CHANGES_REQUESTED** [review]
Hi @KABIKA681, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

## To Highlight
_Good job using flex and grid to layout your application, well done._ 

 ## Required Changes ♻️

- [x] Please consider what the previous reviewer said about writing commit messages. You can visit this [link](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md) to see how you can write professional commit messages 

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Paskab012** [comment]
## Additional comment.
Dear Reviewer, Please use `Chrome` as a browser to view the desktop and mobile version of my project. Other browsers are not displaying it well. This is leading to false comments from reviewers due to that, case of the second reviewer. @medaminedev66.

**Review by rustam-tolipov — APPROVED** [review]
## Hi @KABIKA681  👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 🎉 🎉 🎉 🎉 

Cheers and Happy coding!👏👏👏

image

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**